app/share/metainfo/org.fdroid.Repomaker.appdata.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id type="desktop">org.fdroid.Repomaker.desktop</id>
```

Comment:
```suggestion
  <id type="desktop">org.fdroid.Repomaker</id>
```

Reply:
Fixed in https://github.com/flathub/flathub/pull/771/commits/30651033054252e9f12437cbe4fe31a69c4dc67b.